web/stories/django-girls-bucaramanga.rst

```diff
@@ -247,25 +247,70 @@ contigo para coordinar los detalles.
    <div class="section-guias">
 
 .. template:: bootstrap3/django-girls-guia
-   :name: Diego Fernando Acosta Ortiz
-   :image: guia-diego.png
+   :name: Andrea Pérez Becerra
+   :image: guia-dennise.png
    :place: Bucaramanga, Colombia
-   :community: Python Bucaramanga
-   :web: http://cusol.uis.edu.co/
-   :github: https://github.com/dacostaortiz/
-   :facebook: https://www.facebook.com/dfacos
+   :email: andrea.perez.be@gmail.com
+
+.. template:: bootstrap3/django-girls-guia
+   :name: Yurley Katterine Sanchez Florez
```

Comment:
Here is the other one and more important.

As this name is too long, it breaks the CSS rules since uses more height that it should. Two possible solutions:

1. increase the height for the `bootstrap3/django-girls-guias`, or
1. remove one of the names of this person